### PR TITLE
Docs/syslog

### PR DESCRIPTION
**Pull request by libronlopes**
add instructions for syslog setup, dmsghttp, autocomplete 
update golang version to 1.17 in instructions

### CODE REVIEW

File: docs/software/skywire/setup/syslog.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    !!! Install 
+        To install syslog-ng, run:
+        ```
+        $ sudo pacman -S syslog-ng
+        ```
+    ---
+
+## Build
+
+In skywire uncomment `replace github.com/skycoin/dmsg => ../dmsg` in go.mod and run
```
Comment (jdknives) [inline docs/software/skywire/setup/syslog.md:21]: Only necessary for now. Can be removed as the correct code will automatically be vendored once we push the PR.

File: docs/software/skywire/setup/syslog.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+$ make build
+```
+Run syslog-ng
+```
+$ sudo syslog-ng -f ./init/syslog-ng.conf -edv
+```
+
+In a new terminal, run
+```
+$ ./skywire-visor -c skywire-config.json --syslog localhost:514
```
Comment (jdknives) [inline docs/software/skywire/setup/syslog.md:36]: Note needs to be added that hostname needs to be used here if this does not work. Or check if we can use the output of `hostname` itself can be used

**Review by jdknives — CHANGES_REQUESTED** [review]

File: docs/software/skywire/setup/syslog.md (hunk 1 above)
Comment (libronlopes) [reply docs/software/skywire/setup/syslog.md:21]: Okay

File: docs/software/skywire/setup/syslog.md (hunk 2 above)
Comment (libronlopes) [reply docs/software/skywire/setup/syslog.md:36]: okay

**Pull request merged**